### ISSUE TITLE
Fix Java deprecations - newInstance()

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#5362]
We have many deprecation warnings (over 150) similar to:
```
[WARNING] /Users/jonesn/IdeaProjects/egeria.maven/open-metadata-implementation/access-services/asset-manager/asset-manager-server/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/converters/GlossaryCategoryConverter.java:[56,37] newInstance() in java.lang.Class has been deprecated
```
This was deprecated as of Java 9

Recommend we do this once we've done #5208

**Comment by davidradl** [comment]
https://docs.oracle.com/javase/9/docs/api/java/lang/Class.html#newInstance--

Looks like the replacement is not present at java 8 - so we probably do not want to do this while we support java 8.

**Comment by planetf1** [comment]
Indeed that's why I'm saying let's do it after #5208

**Comment by planetf1** [comment]
We are now on Java 11 in master, so this is doable now.

**Comment by mandy-chessell** [comment]
Is this resolved?

**Comment by planetf1** [comment]
Looks as if this was fixed in July via 46fec31f50c
Can close
